spec.emu

```diff
@@ -266,8 +266,8 @@ contributors: Ujjwal Sharma, Younies Mahmoud
           1. Let _displaySlot_ be the Display Slot value.
           1. Let _valueSlot_ be the Value Slot value.
           1. Let _unit_ be the Unit value.
-          1. Let _style_ be the current value of the _styleSlot_ slot of _durationFormat_.
-          1. Let _nextStyle_ be the current value of the _nextStyleSlot_ slot of _durationFormat_.
+          1. Let _style_ be _durationFormat_.[[&lt;_styleSlot_&gt;]].
```

Comment:
I did it this way previously, but @bakkot asked me to switch. I don't mind either, but let's make sure we're all onboard with this style (fwiw, it's commonly used across 402).

Reply:
Did I? I don't recall. Given that this will go in 402 I'm fine with matching the prevailing editorial conventions of 402.